challenge.md

```diff
@@ -6,19 +6,29 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+git clone <ssh url>
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+git log
+
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
 
+git show <commit id>
+
 4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What you we do? What git command would one use?
 
+git checkout -b <branchName>
+
 
 5. I want to add a file to the staging area. What git commit would I use?
 
+git add <fileName>
 
 6. I've been working on a branch, add-directories, to implement a feature. The feature is done and you need to add your changes into the master branch. What git command should I use?
 
+git merge master
```

Comment:
Not quite. `git merge {branch}` takes commits _from_ the given branch and adds them _to_ your current location in the repo. To merge changes from `add-directories` to `master`, we would need to be on `master` (we can get there by running `git checkout master`) and then run `git merge add-directories`.